### ISSUE TITLE
ensure there is a combobox example to support https://github.com/w3c/aria/pull/1225

### ISSUE THREAD

**Issue by jnurthen** [w3c/aria-practices#1476]
I think there is but dotting the i's and crossing the t's...
Please close if appropriate

**Comment by spectranaut** [comment]
There are two main changes that seems as if they need to be reflected in the examples:

1. If the combobox implementation includes a button to open the dropdown, the button is a button element and is not a child of the combobox element. 

This is reflected in all examples with a button, so there is not work to be done here.

2. The value of the combobox is either (1) the same as the value of the element with combobox role (such as the value of the input) or (2) if that element doesn't have a value then it is the "name" of the element calculated as if the element is a button.

For all combobox examples that use an input as the combobox element, the example makes it clear the value of the input is the value of the combobox. However, it could be made more clear that the value of the [Select-Only Combobox](https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html) is NOT set according to "aria-selected" from the listbox but from the calculation of the combobox element's name alone. It's hard to imagine someone following the design pattern of combobox in a way that makes it so the selected value is not visible and a direct descendant of the combobox in the DOM, but it is technically possible...

@mcking65 do you think this last point should be worded more specifically in the example or do you think we can consider this issue closable?

**Comment by spectranaut** [comment]
What is the conclusion here? Should there be a change to APG? Did someone volunteer to make it? :)

**Comment by jnurthen** [comment]
Yes - we want a change but don't believe it is required for 1.2 to go to PR. No one volunteered to make the change at this point but the title of this issue is updated to reflect the change we would like to make.
